src/main/resources/db/migration/V1.1__Initial_migration.sql

```diff
@@ -0,0 +1,29 @@
+CREATE TABLE characters (
+  id SERIAL PRIMARY KEY,
+  name VARCHAR(100)
+);
+
+CREATE TABLE heroes (
+  id SERIAL PRIMARY KEY,
+  money INTEGER,
+  character INTEGER REFERENCES characters(id)
+);
+
+CREATE TABLE enemies (
+  id SERIAL PRIMARY KEY,
+  character INTEGER REFERENCES characters(id)
+);
+
+CREATE TABLE attributes (
+   id SERIAL PRIMARY KEY,
+   name VARCHAR(100)
+);
+
+INSERT INTO attributes (name) VALUES ('Live'), ('Strength'), ('Mana');
+
+CREATE TABLE character_attributes (
+  id SERIAL PRIMARY KEY,
+  value INTEGER,
+  character INTEGER REFERENCES characters(id),
+  attribute INTEGER REFERENCES attributes(id)
+);
```

Comment:
When you create a new column in postgresql by default is going to be nullable unless you specify the contrary, and I don't think you want to have `characters` with a nullable `name`

More information here: https://www.postgresql.org/docs/current/ddl-default.html

Reply:
Also, in the case that you want nullable columns, you have to handle the case that they are null in the code, throwing an exception or asking if it's null to do something else instead